WolvenKit.Common/RED4/CR2W/JSON/PrimitiveConverter.cs

```diff
@@ -2602,10 +2631,19 @@ public void End()
 
     public void AddReference(string referenceId, T value)
     {
+        try
+        {
+            ReferenceIdToObjectMap[referenceId] = value;
+        }
+        catch (Exception ex)
+        {
+            throw new JsonException($"{ex}");
+        }
+/*
```

Comment:
Why? The value shouldn't be changed after its set, so TryAdd is better.
If its changed probably something other is broken.

Reply:
tried a bunch of things on that one, it seems the issue is with duplicates,
that's the only thing i could get working somehow

Reply:
RefId/BufferId should never be duplicated. So either the Writer is broken or its an user error (modified by hand). Anyway that shouldn't be solved here